sdfg_utils.js

```diff
@@ -634,7 +646,7 @@ function memlet_tree_complete(root_graph) {
         for (const mt of all_memlet_trees) {
             for (const edge of tree) {
                 if (mt.has(edge)) {
-                    mt.add(...tree);
+                    tree.forEach(e => mt.add(e));
```

Comment:
could you explain this change here?

Reply:
The 'add' method of a set takes only one argument so that only one element gets inserted with the spread operator, which might result in incomplete memlet trees